crates/publish/Cargo.toml

```diff
@@ -7,6 +7,8 @@ edition = { workspace = true }
 [dependencies]
 anyhow = "1.0"
 bindle = { workspace = true }
+base64 = "0.21.0"
+dkregistry = { git = "https://github.com/camallo/dkregistry-rs", rev = "37acecb4b8139dd1b1cc83795442f94f90e1ffc5" }
```

Comment:
The latest published version of this crate uses Tokio 0.18, which makes it incompatible with our existing setup.

The functionality used from that crate is minimal (only making an authenticated request to the registry using the credentials, to validate them). If we consider the git source a blocker, we can:

a) remove the functionality altogether — I'm opposed to this, since it catches user errors early
b) fork the repo and publish a new version ourselves
c) update the existing OCI client library to do this instead.

I'm in favor of c).